arangod/Graph/Enumerators/OneSidedEnumerator.cpp

```diff
@@ -121,11 +121,11 @@ auto OneSidedEnumerator<Configuration>::computeNeighbourhoodOfNextVertex()
 
     // Will throw all network errors here
     std::vector<Step*> preparedEnds = std::move(futureEnds.get());
-
     TRI_ASSERT(preparedEnds.size() != 0);
     TRI_ASSERT(_queue.firstIsVertexFetched());
   }
 
+  TRI_ASSERT(!_queue.isEmpty());
```

Comment:
minor comment done in backport: https://github.com/arangodb/arangodb/pull/17546#discussion_r1019013879